### PR TITLE
CT-2562 Fix roles on create case link and remove other button

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Self-review checklist

* [x] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Screenshots
n/a no changes to f/e, just Create case not shown to non/manager and bmt
 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2562
 
### Manual testing instructions
Log in as manager, you should see create case in header, and a green button
Log in as anyone else, you should see no create case in header and no button.

### CODE REVIEW

**Review by Polsonby — COMMENTED** [review]
LGTM! Presumably we already amended all the tests to use the new one which is why no tests changed in this PR?

I have serious reservations about removing the button, though, I pinged Alex to mention it. I don't see why adding it in the main nav means we can't also have it on the page, I think it's really good in the current location. Anyway, happy to approve PR if we're really going to get rid of it :)

File: config/settings.yml [hunk 1]
```diff
@@ -113,6 +113,8 @@ global_navigation:
   pages:
     create_case:
       path: /cases/new
+
```
Comment (Polsonby) [inline config/settings.yml]: Probably don't need this whitespace, maybe remove?

Comment (digitalali-moj) [reply config/settings.yml]: yes, removed, thanks

**Review by Polsonby — APPROVED** [review]
LGTM :) 👍

**Pull request merged**